playbook/appherd/910_add_cron_exports.yml

```diff
@@ -78,5 +78,13 @@
       cd {{ install_root }}/{{ project_name }}/
       echo "$(date --iso-8601=seconds) Active_BBAPI_Applications:$({{ cf_app_py_virtual_env }}/bin/python3 manage.py dumpdata dot_ext.application --indent 2 |grep \"active\": | wc -l ) " >>{{ cf_app_log_dir }}/info.log
 
+  - name: "Export Application List to info.log "
+    become_user: "{{ remote_admin_account }}"
+    become: yes
+    run_once: yes
+    shell: |
+      source {{ cf_app_py_virtual_env }}/bin/activate
+      cd {{ install_root }}/{{ project_name }}/
+      {{ cf_app_py_virtual_env }}/bin/python3 manage.py model2csv --application dot_ext --model application --add_table_name True | sed '/WARNING:/d' | cut -d ',' -f 1,2,3,7,22,23 | sed 's/dot_ext.application/dot_ext.application_info/' >>{{ cf_app_log_dir }}/info.log
```

Comment:
This looks good to me. I'm wondering if it is possible to write a `model2json` command and have Splunk ingest that. Not sure what's required for Splunk to ingest/understand json log files, though.

Reply:
I have found it harder than it ought to be to manipulate the json in splunk.
We also have to get the splunk team to model the data and setup the ingestion.
The info log is captured by splunk and I have worked out searches that can extract the relevant messages from the info.log